executor/distsql.go

```diff
@@ -572,7 +587,11 @@ func (e *IndexLookUpExecutor) buildTableReader(ctx context.Context, handles []kv
 		plans:          e.tblPlans,
 	}
 	tableReaderExec.buildVirtualColumnInfo()
+	rpcStart := time.Now()
 	tableReader, err := e.dataReaderBuilder.buildTableReaderFromHandles(ctx, tableReaderExec, handles, true)
+	if e.stats != nil {
+		recordIndexLookUpRuntimeStats(e.stats.tableTask, time.Since(rpcStart))
+	}
```

Comment:
The time of `e.dataReaderBuilder.buildTableReaderFromHandles` doesn't mean the RPC time, it is the build table reader time, actually this time is useless. 

If you want to count the task num, better put this in the index worker when it to send task, this can also avoid data race.